Ext4Pkg/Ext4Dxe/Ext4Dxe.h

```diff
@@ -797,10 +868,10 @@ Ext4FileIsReg (
            it's a regular file or a directory, since most other file types
            don't make sense under UEFI.
 **/
-#define Ext4FileIsOpenable(File)  (Ext4FileIsReg(File) || Ext4FileIsDir(File))
+#define Ext4FileIsOpenable(File)  (Ext4FileIsReg (File) || Ext4FileIsDir (File) || Ext4FileIsSymlink (File))
 
 #define EXT4_INODE_HAS_FIELD(Inode, Field)                                     \
-  (Inode->i_extra_isize + EXT4_GOOD_OLD_INODE_SIZE >=                          \
+  (Inode->i_extra_isize + (UINTN) EXT4_GOOD_OLD_INODE_SIZE >=                          \

```

Comment:
Maybe we should make macros generally unsigned? I'm not sure because it could also cause the reverse problem, but we hardly ever deal with signed ints.

Reply:
Done in https://github.com/acidanthera/audk/pull/11/commits/8a7d564e43e35ac47372792a0f4fab3adc4dae2e

Reply:
I'm not sure about setting other constants to be unsigned mindlessly, so changes the only one which broke release build in past